src/app/App.tsx

```diff
@@ -1,15 +1,17 @@
 import React, { useState } from 'react';
+import styled from 'styled-components';
 import Card from './components/Card/Card';
 import Form from './components/Form/Form';
+import Button from './components/Button/Button';
 
 type Pupil = {
   name: string;
   evaluations: string[];
 };
 
-function App(): JSX.Element {
+export default function App(): JSX.Element {
   const [pupils, setPupils] = useState<Pupil[]>([]);
-  console.log(pupils);
+  const [displayState, setDisplayState] = useState('');
```

Comment:
It seems like you only have 2 states here, so I would rather work with true/false: 

```suggestion
  const [isFormShown, setIsFormShown] = useState(false);
```